test/auro-datetime.test.js

```diff
@@ -20,6 +20,42 @@ describe('auro-datetime', () => {
     await expect(root.innerHTML).to.equal(`<!---->\n      <slot name="pre"></slot>\n      Tue, Sep 22, 2020\n      <slot name="post"></slot>\n      <slot></slot>\n    <!---->`);
   });
 
+  it('auro-date converts UTC time', async () => {
+    const el = await fixture(html`
+      <auro-datetime type="time" utc="2020-09-22T01:38:22Z"></auro-datetime>
+    `);
+
+    const root = el.shadowRoot;
+    await expect(root.innerHTML).to.equal(`<!---->\n      <slot name="pre"></slot>\n      1:38 am\n      <slot name="post"></slot>\n      <slot></slot>\n    <!---->`);
```

Comment:
These assertions make the tests brittle if we ever add/remove slots or newlines. Could we do something like this instead?

```js
await expect(root.innerHTML).to.include('1:38 am')
```

Another option would be wrapping the datetime in a span and assert on the contents of that span directly.

Reply:
This would make the intent of the test a lot clearer.